apps/client/src/app/components/holding-detail-dialog/holding-detail-dialog.html

```diff
@@ -412,11 +412,10 @@
     </mat-tab-group>
 
     <gf-tags-selector
+      [formControl]="holdingForm.get('tags')"
       [hasPermissionToCreateTag]="hasPermissionToCreateOwnTag"
       [readonly]="!data.hasPermissionToUpdateOrder"
-      [tags]="activityForm.get('tags')?.value"
       [tagsAvailable]="tagsAvailable"
-      (tagsChanged)="onTagsChanged($event)"
```

Comment:
Could you remove the `tagsChanged` output in the tags selector component as well?